### PR TITLE
Footer Upgrade

### PR DESCRIPTION
**Pull request by akshaaatt**
Upgrade the footer section to club everything important regarding ListenBrainz and MetaBrainz.

image

image

### CODE REVIEW

File: listenbrainz/webserver/templates/base.html [hunk 1]
```diff
@@ -4,60 +4,139 @@
...
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://metabrainz.org/" target="_blank" rel="noopener noreferrer" >MetaBrainz</a>
+            </li>
+
+          </ul>
+        </div>
+        <div class="col-sm-12 col-md-3 section-md-t3">
+          <h3 class="w-title-a text-brand">Fellow Projects</h3>
+          <ul class="list-unstyled">
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/>  <img src="{{ url_for('static', filename='img/meb-icons/MusicBrainz.svg') }}" width="24" height="24" alt="image"/> <a href="https://musicbrainz.org/" target="_blank" rel="noopener noreferrer" >MusicBrainz</a>
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/base.html:94]: In complement to what I said above about `alt` descriptions, here for the project logos images we want to change that to `alt="MusicBrainz logo` etc.

File: listenbrainz/webserver/templates/base.html [hunk 2]
```diff
@@ -4,60 +4,139 @@
...
-      {%- endblock -%}
-    </div>
+    {%- block content -%}
+    <!-- Page content -->
     {%- endblock -%}
+  </div>
+  {%- endblock -%}
+
+  {%- block footer -%}
+  <section class="footer">
```
Comment (MonkeyDo) [inline listenbrainz/webserver/templates/base.html:42]: The footer butts up right next to the content of the page*, I think maybe adding a `margin-top: 2em` or something like that might be good.

Currently:
image

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
Overall I think this looks a lot nicer than the previous footer, so it's a big :+1: from me !

File: listenbrainz/webserver/templates/base.html (hunk 2 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:42]: Done

File: listenbrainz/webserver/templates/base.html [hunk 3]
```diff
@@ -4,60 +4,139 @@
...
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-12 col-md-6">
+          <h3>
+            <img src="{{ url_for('static', filename='img/logo_big.svg') }}" width="52" alt="image"/>
+            <span class="color-purple">Listen</span><span class="color-orange">Brainz</span>
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:48]: Done

File: listenbrainz/webserver/templates/base.html [hunk 4]
```diff
@@ -4,60 +4,139 @@
...
+          </h3>
+          <br/>
+          <p class="color-gray">
+            keeps tracks of what music you listen to and provides you with insights into your listening habits. We're completely open-source and publish our data as open data.
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:52]: Done

File: listenbrainz/webserver/templates/base.html (hunk 1 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:94]: Done

File: listenbrainz/webserver/templates/base.html [hunk 5]
```diff
@@ -4,60 +4,139 @@
...
+        </div>
+        <br/>
+        <div class="col-sm-12 col-md-3 section-md-t3">
+          <h3 class="w-title-a text-brand">Useful Links</h3>
+          <ul class="list-unstyled">
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://metabrainz.org/donate" target="_blank" rel="noopener noreferrer" >Donate</a>
+            </li>
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="24" height="24" alt="image"/> <a href="https://wiki.musicbrainz.org/Main_Page" target="_blank" rel="noopener noreferrer" >Wiki</a>
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:73]: Done

File: listenbrainz/webserver/templates/base.html [hunk 6]
```diff
@@ -4,60 +4,139 @@
...
+        <div class="col-sm-12 col-md-6">
+          <h3>
+            <img src="{{ url_for('static', filename='img/logo_big.svg') }}" width="52" alt="image"/>
+            <span class="color-purple">Listen</span><span class="color-orange">Brainz</span>
+          </h3>
+          <br/>
+          <p class="color-gray">
+            keeps tracks of what music you listen to and provides you with insights into your listening habits. We're completely open-source and publish our data as open data.
+
+            You can use ListenBrainz to track your music listening habits and share your taste with others using our visualizations. We also have an API if you want to do more with our data.
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html]: Done

File: listenbrainz/webserver/templates/base.html [hunk 7]
```diff
@@ -4,60 +4,139 @@
...
+            <li class="color-a">
+              <span class="color-gray">IRC: </span> <a href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz" target="_blank" rel="noopener noreferrer" > #metabrainz</a>
+            </li>
+            <li class="color-a" >
+              <span class="color-gray">Email: </span> <a href="mailto:support@metabrainz.org">support@metabrainz.org </a>
+            </li>
+          </ul>
+        </div>
+        <br/>
+        <div class="col-sm-12 col-md-3 section-md-t3">
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html]: Done

File: listenbrainz/webserver/templates/base.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html]: Done

File: listenbrainz/webserver/templates/base.html (hunk 1 above)
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html]: Done

File: listenbrainz/webserver/templates/base.html [hunk 8]
```diff
@@ -4,60 +4,137 @@
...
+            <li class="item-list-a">
+              <img src="{{ url_for('static', filename='img/icons/angle_double_right_icon.svg') }}" width="18" height="18"/> <img src="{{ url_for('static', filename='img/meb-icons/CoverArtArchive.svg') }}" width="18" height="18" alt="CoverArtArchive"/> <a href="https://coverartarchive.org" target="_blank" rel="noopener noreferrer" >Cover Art Archive</a>
+            </li>
+          </ul>
+        </div>
       </div>
-    {%- endblock -%}
-  </div>
+      <div class="row center-p">
+        <div class="col-md-3 d-none d-md-block hidden-xs">
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/templates/base.html:119]: Thanks for bringing this up! I have added the Contribution link and the bug tracker link to the `Useful Links` section which is available for the mobile layout only. On bigger screens, they are showcased as shown in the screenshot.

**Review by MonkeyDo — APPROVED** [review]
Great, this looks awesome, nice work! :tada:

**Pull request merged**